src/topology/unit_test.rs

```diff
@@ -597,13 +611,73 @@ mod tests {
         )
         .unwrap();
 
-        let mut tests = build_unit_tests(&config).unwrap();
+        let mut tests = build_unit_tests(&mut config).unwrap();
+        assert_eq!(tests[0].run().1, Vec::<String>::new());
+    }
+
+    #[test]
+    fn test_swimlanes() {
+        let mut config: Config = toml::from_str(
+            r#"
+[transforms.foo]
```

Comment:
I'm curious if you think this is better served as a behavior test? https://github.com/timberio/vector/pull/1781